internal/controller/vmfilerestore_controller_test.go

```
@@ -0,0 +1,897 @@
+package controller
+
+import (
+	"context"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/go-logr/logr"
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	oadpv1alpha1 "github.com/openshift/oadp-operator/api/v1alpha1"
```

golangci-lint (2.5.0)

[error] File is not properly formatted

(gci)